debian/mariadb-server-10.4.mysql-server.logrotate

```diff
@@ -11,8 +11,8 @@
 	sharedscripts
 	postrotate
           test -x /usr/bin/mysqladmin || exit 0
-          if [ -f `my_print_defaults --mysqld | grep -oP "pid-file=\K[^$]+"` ]; then
-            # If this fails, check debian.conf!
+	  # check if server is running
+          if mysqladmin ping > /dev/null 2>&1; then
             mysqladmin --defaults-file=/etc/mysql/debian.cnf --local flush-error-log \
```

Comment:
I'd simply remove the whole check and just run
```
mysqladmin flush-engine-log flush-general-log flush-slow-log 2>&1 >/dev/null || :
```
if `mysqladmin ping` will fail to connect then `mysqladmin flush` can fail just the same, no need to run it twice.

Reply:
Do you want to fix it yourself or you'd prefer me to do this simple change?

Reply:
If you just drop any error messages you will not get noticed if the flush-log command fails for any other reason (e.g. file system full). I would rather prefer my version: Making sure the server can listen to commands and then telling it what to do.